typescript/packages/agents-m365copilot-beta/package.json

```diff
@@ -10,11 +10,11 @@
 	"homepage": "https://github.com/microsoft/Agents-M365Copilot/blob/main/typescript/README.md",
 	"license": "MIT",
 	"main": "dist/index.node.js",
-	"types": "dist/index.d.ts",
+	"types": "dist/generated/index.d.ts",
 	"browser": "dist/index.browser.js",
 	"exports": {
 		".": {
-			"types": "./dist/index.d.ts",
+			"types": "./dist/generated/index.d.ts",
```

Comment:
could we instead add an index file at the root to re-export?
this way if we need to add handcrafted code at any point, we'll already be setup for it.